### DISCUSSION TITLE
Change Bibliography Title

### DISCUSSION THREAD

**Discussion by catafjs** [joaomlourenco/novathesis#411] [Q&A] [answered]
Hello, I'm from NOVA IMS and I'm finishing writing my thesis in Version 2022-01-14 [6.9.2].

My supervisor asked me to change the title of the Bibliography to "References".

I tried to do it in the "novathesis.cls" file by changing "bib" to "bib[title=References]" as indicated in comment. However, it didn't work out. Is there another alternative?
![image](https://user-images.githubusercontent.com/125210627/218327092-309b35e8-60d9-42c9-ae0b-2b4fc5ac1f7a.png)

**Comment by joaomlourenco** [comment, marked as answer]
In line 877, replace `bib;` with `\printbibliography[title=References];`

Please confirm if it worked for you. Thanks.

**Reply by catafjs** [reply]
It worked!! Thank you so much!

**Reply by joseluisgomes** [reply]
Thank you so much!
